src/components/entity-info.component.js

```diff
@@ -25,7 +25,7 @@ const StyledListItem = styled(ListItem).attrs({
   },
   underlayColor: props => (props.unknown ? null : colors.greyLight),
   hideChevron: props => props.unknown,
-});
+})``;
```

Comment:
?

Reply:
@andrewda It is just my [last commit](https://github.com/gitpoint/git-point/pull/612/commits/e00addbaae91defb8e96ea9fdf8e13ff3f0b967b), whose purpose is reverting changes. I removed them by mistake, because `stylelint` complained about empty blocks, as the updated PR description mentioned. But we can't simply do that. "fn``" is different with "fn".

Reply:
Ah ok yea, didn't see that that was a revert commit.